datastore/google/cloud/datastore/_http.py

```diff
@@ -448,11 +448,14 @@ def rollback(self, project, transaction_id):
         :type transaction_id: str
         :param transaction_id: The transaction ID returned from
                                :meth:`begin_transaction`.
+
+        :rtype: :class:`.datastore_pb2.RollbackResponse`
+        :returns: The returned protobuf response object.
         """
         request = _datastore_pb2.RollbackRequest()
         request.transaction = transaction_id
-        # Nothing to do with this response, so just execute the method.
-        self._datastore_api.rollback(project, request)
+        # Response is empty (i.e. no fields) but we return it anyway.
```

Comment:
Because the point of this PR is to honor the protobuf response. 

Also, it's not "nothing", just a [message][1] with no fields. Not sure why the datastore folks defined their own version of [`Empty`][2].

[1]: https://github.com/googleapis/googleapis/blob/8c5b4c491603e0b23c7d013941b1354d0b849caf/google/datastore/v1/datastore.proto#L158
[2]: https://github.com/google/protobuf/blob/008ff033ef3308512c1da2b7cdadf50a6296251a/src/google/protobuf/empty.proto#L52